packages/perseus/src/components/graph.tsx

```diff
@@ -1,10 +1,11 @@
-/* eslint-disable react/forbid-prop-types, react/no-unsafe, react/sort-comp */
+/* eslint-disable react/no-unsafe, react/sort-comp */
```

Comment:
Glad that we were able to remove a rule exception. Fixing `react/sort-comp` is pretty low-hanging fruit; I'm wondering if we should do a quick pass sometime soon to fix those errors.